src/client/datascience/interactive-common/interactiveBase.ts

```diff
@@ -1401,7 +1401,7 @@ export abstract class InteractiveBase extends WebViewHost<IInteractiveWindowMapp
             const updatedHeights = payload as { containerHeight: number; gridHeight: number };
             const uri = await this.getOwningResource(); // Get file name
 
-            if (!uri) {
+            if (!uri || uri.scheme === 'untitled') {
```

Comment:
```suggestion
            if (!uri || isUntitledFile(uri)) {
```

`isUntitledFile` can be found in `nativeEditorStorage.ts`